### ISSUE TITLE
`grafana_image` should complain when there are multiple dashboards with the same name

### ISSUE THREAD

**Issue by achew22** [etsy/rules_grafana#10]
When there are multiple `dashboards` included that have `srcs` with the same name, you get the warning:

```
Duplicate file in archive: ./var/lib/grafana/dashboards/dash.generated.json, picking first occurrence
```

I believe this should be an error instead of a warning since it is silently omitting a dashboard. Also, is it possible to have the `py_dashboards` rule include something like a prefix on generated files so that it is more difficult to run into this sort of problem?

Thanks so much! The project is really great.
